docs/content/users/configuration/config.md

```diff
@@ -562,7 +562,7 @@ Example: `webimage_extra_packages: [php-yaml, php-bcmath]` will add the `php-yam
 
 ## `webserver_type`
 
-Whether Xdebug should be enabled for [step debugging](../debugging-profiling/step-debugging.md) or [profiling](../debugging-profiling/xdebug-profiling.md).
+Which available web server should be used.
```

Comment:
```suggestion
Which available [web server type](../extend/customization-extendibility.md/#changing-web-server-type
) should be used (`nginx-fpm` (default) or `apache-fpm`).
```

Reply:
@rfay The link seems useful, but the parenthetical is redundant with the details that immediately follow:

![Rendered output highlighting redundant bits](https://user-images.githubusercontent.com/2488775/219248477-500902b1-c755-48f3-88e3-2f34c04ab68f.png)

Reply:
That's the problem with tiny changes, can't see the context!

Reply:
Thanks to both of you.  I committed with the link, but without the repetition.